server/sockets/cohort.js

```diff
@@ -2,6 +2,29 @@ const { CohortActionTypes } = require('../common/variables');
 const Cohort = require('../models/cohort.model');
 const { responseWithCohort } = require('../common/utils');
 
+const emitCohortMetaData = (cohortId, clients, socket) => {
+  Cohort.findById(cohortId)
+    .select('_id isArchived createdAt name description memberIds')
+    .lean()
+    .exec()
+    .then(doc => {
+      if (doc) {
+        const { memberIds } = doc;
+        const cohort = responseWithCohort(doc);
+
+        memberIds.forEach(id => {
+          const memberId = id.toString();
+
+          if (clients.has(memberId)) {
+            socket.broadcast
+              .to(clients.get(memberId))
+              .emit(CohortActionTypes.CREATE_SUCCESS, cohort);
+          }
+        });
+      }
+    });
+};
+
```

Comment:
create /helperes directory within sockets directory, and put it in index.js file and import it here 

![Zrzut ekranu 2019-07-18 o 07 55 40](https://user-images.githubusercontent.com/27632432/61432695-7b61a800-a931-11e9-8f22-44fa48be9ca7.png)